### ISSUE TITLE
Cannot run ansible-navigator using Python3.8 on osx

### ISSUE THREAD

**Issue by clevesque** [cloudera-labs/cloudera-deploy#128]
tested on mac osx...

ansible-navigator using a Py3.8 env is unable to parse ansible-navigator.yml files as in our public-cloud/aws/ examples

when used with Py3.11 env, it was able to use the ansible-navigator.yml setting file

did not test any other versions of Python

**Comment by wmudge** [comment]
https://ansible.readthedocs.io/projects/navigator/installation/#requirements-macos suggests you will need `3.9` or higher.  This link is called out with an **Important** admonition, but perhaps we should make it even more clear by forcing a Python version > 3.8 in the `venv` example?

```
python3.9 -m venv ~/cdp-navigator; source ~/cdp-navigator/bin/activate;
```

**Comment by wmudge** [comment]
Closed by #132
